docs/developer/deploy/index.rst

```diff
@@ -0,0 +1,6 @@
+Deploy new MetalK8s image
```

Comment:
Typo in the commit message: `s/Mtelak8s/Metalk8s/`

Reply:
done